### ISSUE TITLE
Binary File Ispector: select focus is overwritten

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#186]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/typedarrays/

The blue border focus doesn't show up, so is presumably overwritten later

**Comment by sarvaje** [comment]
@dstorey this is because of an style in EdgePortal that overwrite the outline property.
@melanierichards @ststimac can we overwrite this style? (at least for this demo)

**Comment by melanierichards** [comment]
@sarvaje where is the overwriting style in Edge Portal? That doesn't seem like it should be happening in the first place. But to answer your question you can make an exception if needed on this demo.

**Comment by sarvaje** [comment]
sorry @melanierichards, the overwrite is in [EdgeCore](https://github.com/MicrosoftEdge/EdgeCore/blob/Indigo/_source/_core/css/controls.css#L70).

**Comment by dstorey** [comment]
@melanierichards @sarvaje this is probably because demo.css restyles the select and file input so the styles are clashing. Best fix would be just to remove that custom styling as it isn't adding any value now we have a house style.

**Comment by melanierichards** [comment]
I agree with @dstorey.
